src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx

```diff
@@ -671,37 +671,30 @@ export class PrebuiltPredictPage extends React.Component<IPrebuiltPredictPagePro
 
             const extendPredictionItem = (key, field) => {
                 const result = {};
+                const updateFieldValueToResult = (item, itemName) => {
+                    if (item.valueObject) {
```

Comment:
suggestion: ideally we should have different function for handling different structured item.valueXxx. But that might be out of this PR's scope IMHO.